tests/bearlib/aspects/SubTest.py

```diff
@@ -1,14 +1,13 @@
-from coalib.bearlib.aspects import Root, aspectclass, TasteError
-from coalib.bearlib.aspects.base import aspectbase
+from coalib.bearlib.aspects import Root, TasteError
+from coalib.bearlib.aspects.meta import isaspect
 
 import pytest
 
 
 class SubAspectTest:
 
     def test_class(self, SubAspect):
-        assert isinstance(SubAspect, aspectclass)
-        assert issubclass(SubAspect, aspectbase)
+        assert isaspect(SubAspect)
```

Comment:
Same as in `RootTest`: Not fully sufficient because `isaspect`'s argument can also be an aspect instance